automated install/basic-install.sh

```diff
@@ -1223,7 +1223,9 @@ chooseBlocklists() {
 
     # In a variable, show the choices available; exit if Cancel is selected
     choices=$("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty) || { printf "  %bCancel was selected, exiting installer%b\\n" "${COL_LIGHT_RED}" "${COL_NC}"; rm "${adlistFile}" ;exit 1; }
-    # For each choice available,
+    # create empty adlist file if no list was selected
+    : > "${adlistFile}"
```

Comment:
Would it work better to touch the file? I've never seen a noop redirect.

Reply:
It is also used in https://github.com/pi-hole/pi-hole/blob/10addc5923c5c43b4488f2d962bde077d677aa7b/gravity.sh#L691 like this. `touch` would also work, I can use that if you like it better.

Reply:
For consistency's sake, I'd say if we're already using it then may as well stick with `: >`

Reply:
Yes, I agree. Keep it as is.